<launch>

    <!-- start the motors -->
    <node name="pololu_driver" pkg="pololu_driver" type="pololu_node.py" output="screen">
        <!-- set topics input and outputs -->
        <param name="port" value="/dev/ttyACM0"/>
        <param name="topic_throttle" value="/motors/throttle"/>
        <param name="srv_switch" value="/motors/switch"/>
    </node>

    <!-- start the pilot -->
    <node name="pilot" pkg="asv_pilot" type="asv_pilot.py" output="screen">
        <!-- set topics input and outputs -->
        <param name="topic_throttle" value="/motors/throttle"/>
        <rosparam command="load" file="$(find asv_pilot)/conf/pid_sim.yaml" />
    </node>

    <node name="nessie_follower" pkg="asv_pilot" type="follower.py" output="screen">
        <param name="topic_position_request" value="pilot/position_req"/>
        <param name="topic_emily_nav" value="nav/nav_sts"/>
        <param name="topic_nessie_nav" value="ds/nav/nav_sts"/>
        <param name="simulation" value="true" type="bool"/>
        <param name="static_latitude" value="0.001"/>
        <param name="static_longitude" value="0.001"/>
    </node>


</launch>
